sky/utils/dag_utils.py

```diff
@@ -64,53 +64,105 @@ def convert_entrypoint_to_dag(entrypoint: Any) -> 'dag_lib.Dag':
     return converted_dag
 
 
-def load_chain_dag_from_yaml(
+def load_dag_from_yaml(
```

Comment:
We can add a TODO here to migrate this two function in the `sky.Dag` class. dont need to implement in this PR though